docs/content/en/docs/workflows/debug.md

```diff
@@ -57,33 +75,53 @@ specified in [Skaffold's global configuration]({{< relref "/docs/design/global-c
     spec: ...
     ```
 
-### Supported Language Runtimes
+## Supported Language Runtimes
 
 Debugging is currently supported for:
-  - Go (runtime ID: `go`)
-  - NodeJS (runtime ID: `nodejs`)
-  - Java and JVM languages (runtime ID: `jvm`)
-  - Python (runtime ID: `python`)
-  - .NET Core (runtime ID: `netcore`)
+  - Go (runtime ID: `go`) using [Delve](https://github.com/go-delve/delve)
+  - NodeJS (runtime ID: `nodejs`) using the NodeJS Inspector (Chrome DevTools)
+  - Java and JVM languages (runtime ID: `jvm`) using JDWP
+  - Python (runtime ID: `python`) using `debugpy` (Debug Adapter Protocol) or `pydevd`
```

Comment:
are all Pythons version supported?

Reply:
nit: if might make sense for supported versions for each runtime (if there is such limitation) be put here or in the langauge specific sections

Reply:
I moved the runtimes lists to the top and added versions where it makes sense.